com.unity.netcode.gameobjects/Runtime/Transports/UTP/UnityTransport.cs

```diff
@@ -1085,13 +1085,22 @@ private void ClearSendQueuesForClientId(ulong clientId)
 
         private void FlushSendQueuesForClientId(ulong clientId)
         {
-            foreach (var kvp in m_SendQueue)
+            // In the event this throws and exception, log it
+            // and allow the invoking method to continue.
+            try
```

Comment:
I don't think catching the exception is the correct course of action here. We should simply check `m_Driver.IsCreated` (perhaps directly in `SendBatchedMessages`) and exit early if it's not created. The problem with the exception is that it will only be thrown when safety checks are enabled (i.e. in the editor). In a player no exception will be thrown and instead we'll end up accessing garbage memory (with failure modes that are less predictable).

Reply:
Thanks for catching that! 👍 
Made the changes.